.github/workflows/phpunit.yaml

```diff
@@ -2,7 +2,11 @@ name: "PHPUnit tests"
 
 on:
     pull_request:
+        branches:
+            - master
     push:
+        branches:
+            - master
```

Comment:
Why only on master? should be checked with every commit?

Reply:
This checks for every commit to the master branch, whether via direct push or a pull request. It just prevents the runner from launching in random branches, which among other things was abused in the past.